test/fixtures/poro.rb

```diff
@@ -16,8 +16,8 @@ def method_missing(meth, *args)
     end
   end
 
-  def cache_key_with_digest
-    "#{cache_key}/#{FILE_DIGEST}"
+  def digest
+    FILE_DIGEST
   end
```

Comment:
looks like it should be a test expectation helper as it's not really a model concern, as I review it.  I don't think the new `digest` is used at all, right?